server/state-selectors-generator/script-generate.js

```diff
@@ -0,0 +1,2 @@
+const stateSelectorsGenerator = require( './' );
+stateSelectorsGenerator.generateIndexFile();
```

Comment:
<img src="https://cloud.githubusercontent.com/assets/77539/22162889/513917ee-df2f-11e6-8097-5306b1160f47.png" width="400px" />

It seems to be already there. 👻